.github/workflows/lint.yml

```
@@ -0,0 +1,14 @@
+name: Lint
+
+on: [push]
+
+jobs:
+  phpcs-lint:
+    runs-on: ubuntu-latest
+    steps:
+        - uses: actions/checkout@v2
+        - name: PHP Lint
+          run: |
+              composer install
+              ./vendor/bin/phpcs
```

I think we can use `php-actions/composer@v6` for the installation step, and if we add a composer script for linting we can run it like so:

```suggestion
        - uses: actions/checkout@v2
        - name: PHP Lint
          uses: php-actions/composer@v6
          with:
              command: lint
```